src/storage/mod.rs

```diff
@@ -1397,28 +1397,60 @@ quick_error! {
 
 pub type Result<T> = ::std::result::Result<T, Error>;
 
-pub fn get_tag_from_header(header: &errorpb::Error) -> &'static str {
+pub enum ErrorHeaderKind {
+    NotLeader,
+    RegionNotFound,
+    KeyNotInRegion,
+    StaleEpoch,
+    ServerIsBusy,
+    StaleCommand,
+    StoreNotMatch,
+    RaftEntryTooLarge,
+    Other,
+}
+
+impl Display for ErrorHeaderKind {
```

Comment:
note: this `Display` impl is supposed to be removed in future, once coprocessor is using static metrics as well.

Reply:
Please add a todo.